crates/service/src/prelude/global/mod.rs

```diff
@@ -88,10 +94,16 @@ pub trait FloatCast: Sized {
     }
 }
 
+pub trait NumRange: Sized {
+    fn max_bound() -> Self;
```

Comment:
Since int8 don't satisfy  the request of num_trait:Float， I replace it with this trait to get max value and mini value.